docs/INSTALLATION.md

```diff
@@ -4,53 +4,26 @@ The C++ language track requires that you have the following software
 installed on your system:
 * a modern C++11 compiler
 * the CMake cross-platform build system
-* the Boost libraries
 
-Microsoft's Visual Studio IDE and Apple's Xcode IDEs combine the compiler
-and the
-native build system into a single IDE (integrated development environment).
-Linux environments typically expose the compiler and native build system
-as separate command-line tools.
+We use [Catch2](https://github.com/catchorg/Catch2) for testing, which is a
+header-only library that is bundled along with the exercises.
 
-### Prerequisite: A Modern C++11 Compiler
+This set of dependencies is very common for C++ projects, so if you have any
+issues setting it up you will find many helpful resources online.
+
+Many IDE's, including Microsoft's Visual Studio, Apple's Xcode, and Jetbrain's
+Clion bundle a compiler and CMake together, so you should only need to install
```

Comment:
Capital 'B', trailing apostrophe, and a capital 'L': `JetBrains' CLion`
But no one is really watching ;)